Libraries/libcar/Sources/Rendition.cpp

```diff
@@ -227,7 +188,9 @@ Load(
                         };
                         slices.push_back(slice);
                 }
-                rendition.slices() = std::move(slices);
+                if (slices.size() > 0) {
```

Comment:
Yes, but this doesn't seem to actually do anything. If the if statement wasn't there, rendition.slices() would still be empty. The only effect seems to avoid the copy?